Annotato/Annotato/Storage/PDFStorageManager.swift

```diff
@@ -2,20 +2,39 @@ import Foundation
 import AnnotatoSharedLibrary
 
 class PDFStorageManager {
-    private var localStorageService: AnnotatoStorageService
+    private var localStorageService = LocalStorage()
     private var remoteStorageService: AnnotatoStorageService
```

Comment:
Lets create two protocols instead of doing this. 

`AnnotatoLocalStorageService` and `AnnotatoRemoteStorageService`